sysutils/daemon/Portfile

```diff
@@ -21,18 +21,19 @@ long_description        Daemon turns other process into daemons. There are many
 homepage                https://libslack.org/daemon/
 master_sites            ${homepage}download/
 
-checksums               rmd160  907a5d09bc405c0a2a23f63d045bdd254423858d \
-                        sha256  bd6fd870ca4761f43f045d72db0f8a0de81a3eac07264bf449b152d7dd899ac0 \
-                        size    466962
+checksums               rmd160  bac3901dab24e01c0dc501be2b61e0d0a5bdd977 \
+                        sha256  fa28859ad341cb0a0b012c11c271814f870482013b49f710600321d379887cd1 \
+                        size    481216
 
 patchfiles              disable-test-pseudo.patch
 
 # Only a script, not a real configure.
 use_configure           yes
-configure.args          --prefix=${prefix} --destdir=${destroot}
+configure.args          --prefix=${prefix} --destdir=${destroot} --disable-mail-test
```

Comment:
what's the purpose of this additional flag; what functionality does it disable and why?

Reply:
It's purpose is to prevent testing a function (in libslack's net module) that sends an email to the current user via the SMTP server on localhost's tcp port 25 (if any). It was brought to my attention that this would be undesirable behaviour when building and testing packages for Debian, so I made it easy to suppress that test. It seemed like a good idea to suppress it for other packaging systems as well. It has no effect on the final daemon binary which doesn't use that function.